components/tenant-mgt/org.wso2.carbon.tenant.keystore.mgt/src/main/java/org/wso2/carbon/keystore/mgt/KeyStoreGenerator.java

```diff
@@ -177,7 +177,7 @@ private X509Certificate generateKeyPair(KeyStore keyStore) throws KeyStoreMgtExc
             KeyPair keyPair = keyPairGenerator.generateKeyPair();
 
             // Common Name and alias for the generated certificate
-            String commonName = "CN=" + tenantDomain + ", OU=None, O=None L=None, C=None";
+            String commonName = "CN=" + tenantDomain + ", OU=None, O=None, L=None, C=None";
```

Comment:
Did you do this change due to any particular error? Trying to understand the relationship with the dependency update and this change.

Reply:
The following error was encountered after the upgrade while creating the tenant keystore. The change has been made to resolve it.

```
Caused by: java.lang.IllegalArgumentException: badly formatted directory string
 	at org.bouncycastle.asn1.x500.style.IETFUtils.nextToken(Unknown Source) ~[bcprov-jdk18on-1.78.1.jar:?]
 	at org.bouncycastle.asn1.x500.style.IETFUtils.addRDN(Unknown Source) ~[bcprov-jdk18on-1.78.1.jar:?]
 	at org.bouncycastle.asn1.x500.style.IETFUtils.addRDNs(Unknown Source) ~[bcprov-jdk18on-1.78.1.jar:?]
 	at org.bouncycastle.asn1.x500.style.IETFUtils.rDNsFromString(Unknown Source) ~[bcprov-jdk18on-1.78.1.jar:?]
 	at org.bouncycastle.asn1.x500.style.BCStyle.fromString(Unknown Source) ~[bcprov-jdk18on-1.78.1.jar:?]
 	at org.bouncycastle.asn1.x500.X500Name.<init>(Unknown Source) ~[bcprov-jdk18on-1.78.1.jar:?]
 	at org.bouncycastle.asn1.x500.X500Name.<init>(Unknown Source) ~[bcprov-jdk18on-1.78.1.jar:?]
 	at org.wso2.carbon.keystore.mgt.KeyStoreGenerator.generateKeyPair(KeyStoreGenerator.java:172) ~[org.wso2.carbon.tenant.keystore.mgt_4.9.10.jar:?]
 	... 67 more

```